src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -57,10 +57,10 @@
     <ProjectReference Include="..\Protocol\Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj" />
 
     <!-- Dlls we don't directly reference but need to include to build the MEF composition -->
-    <ProjectReference Include="..\..\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.Features.csproj" />
+    <ProjectReference Include="..\..\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.Features.csproj" ReferenceOutputAssembly="false" />
```

Comment:
This one doesn't strictly need this since if we did reference something out of this project, there's no real harm there. (Unlike the RazorCompiler case where that's absolutely not good.)